docs/components/custom-dns-server.rst

```diff
@@ -26,8 +34,17 @@ to
 ::
 
     dns_server:
-        - "1.1.1.1"
-        - "1.0.0.1"
+        - "9.9.9.9"
+        - "149.112.112.112"
+
+
+.. version:: 3.0.0
+
+Since :version:`3.0.0` it is possible to assign a specific port to use with the 
+DNS-Server.
+
+.. example::
```

Comment:
the `.. example::` Is not known by read the docs...  Please report to https://sphinx-rtd-theme.readthedocs.io/en/stable/demo/demo.html#admonitions

Reply:
Based on the https://sphinx-rtd-theme.readthedocs.io/en/stable/demo/demo.html#admonitions The use of `hint` seems like a better chois of `wording` for this example box. Alternatively it should be converted into a [Code Blocks](https://sphinx-rtd-theme.readthedocs.io/en/stable/demo/demo.html#code-blocks)

```
.. code-block:: shell
    --dns 95.216.209.53:53 116.203.32.67:53 9.9.9.9:853
```